### PR TITLE
DDCE-5029: Updating optional form fields to filter smart apostrophes

### PR DESCRIPTION
**Pull request by adageorg**

### CODE REVIEW

File: app/forms/InternationalAddressFormProvider.scala [hunk 1]
```diff
@@ -46,9 +46,8 @@ class InternationalAddressFormProvider @Inject() extends Mappings {
               maxLength(maximumLength, "internationalAddress.error.line2.length"),
               regexp(Validation.addressLineRegex, "internationalAddress.error.line2.invalidCharacters")
             )),
-      "line3" ->
-        optional(Forms.text
-          .transform(trimWhitespace, identity[String])
+  "line3" ->
```
Comment (MRBeresford) [inline app/forms/InternationalAddressFormProvider.scala:49]: odd formatting?

**Review by MRBeresford — CHANGES_REQUESTED** [review]
one formatting question

File: app/forms/InternationalAddressFormProvider.scala (hunk 1 above)
Comment (adageorg) [reply app/forms/InternationalAddressFormProvider.scala:49]: it is a bit, isn't it?  I shall fix.

**Review by MRBeresford — APPROVED** [review]
LGTM

**Pull request merged**